NewEvaluation.css

```diff
@@ -190,9 +204,22 @@ input[type=submit]{
 	float: right;
 	box-shadow: 5px 5px 5px #888888;
 }
-.spacing{
-	padding-top: 25vh;
-}
-.littleSpacing{
-	padding-top: 10vh;
+
+
+
+
+
+/*-----------------   Footer -----------------*/
+
+
+.BottomFooter{
+	bottom:0;
+    position:fixed;
+    z-index:150;
+    _position:absolute;
+    _top:expression(eval(document.documentElement.scrollTop+
```

Comment:
I used this piece of code more or less because I had issues with placing the footer the way I desired. (sorry for late responses )